src/openscm_runner/run.py

```diff
@@ -76,6 +76,8 @@ def run(
             runner = MAGICC7()
         elif climate_model.upper() == "FAIR":  # allow various capitalisations
             runner = FAIR()
+        elif climate_model.upper() == "FAIR2":  # allow various capitalisations
```

Comment:
Any thoughts on how you want to do naming? We could have it so FaIR always requires a version (throws error otherwise) and then use a regex to decide if we're looking at FaIR1X or FaIR2+?

Reply:
Or, thinking about it more, will it even be possible to install both FaIR1.X and FaIR2 at the same time given their namespace clash?